spec/rails_helper.rb

```diff
@@ -89,3 +89,7 @@
     with.library :rails
   end
 end
+
+Kaminari.configure do |config|
```

Comment:
Might be nice to add a comment explaining why this overrides the default only affects tests.